inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java

```diff
@@ -240,40 +245,38 @@ public List<AuditVO> listByCondition(AuditRequest request) throws Exception {
     }
 
     private List<String> getAuditIds(String groupId, String streamId, String sinkNodeType) {
-        List<String> auditIds = LoginUserUtils.getLoginUser().getRoles().contains(UserRoleCode.ADMIN)
-                ? new ArrayList<>(auditIdListForAdmin)
-                : new ArrayList<>(auditIdListForUser);
+        Set<String> auditSet = LoginUserUtils.getLoginUser().getRoles().contains(UserRoleCode.ADMIN)
+                ? new HashSet<>(auditIdListForAdmin)
+                : new HashSet<>(auditIdListForUser);
+
+        Object fixedObj = new Object();
+        Map<String, Object> auditIdMap = auditSet.stream().collect(Collectors.toMap(x -> x, x -> fixedObj));
```

Comment:
Here, we only need to use set to obtain ids instead of map to encapsulate other information.

Reply:
OK, the key here is just an immutable string, so we can directly use the add method of the Set.